l10n/en/products/vpn/platforms/desktop.ftl

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
```

Comment:
This is not working yet, but I assume it will soon.

Reply:
Yup. Once this PR merges!